package.json

```diff
@@ -1,6 +1,28 @@
 {
-  "private": true,
```

Comment:
we doesn't need this file to keep logic anymore with husky since v.4 -> everything is in the git hooks themselves

Reply:
ok, I will check

Reply:
I think we should keep general pre-commit or pre-push hooks (in case we don't use checks on CI.) But also I think checks on CI are MUST. So, your config is more general for template.